File: CS/AnimationExample/MainPage.cs
using DevExpress.XtraCharts;
using System;
using System.Windows.Forms;

namespace AnimationExample {
    public partial class MainPage : DevExpress.XtraEditors.XtraForm {
        Series series { get { return chartControl.Series[0]; } }
        FunnelSeriesView view { get { return series.View as FunnelSeriesView; } }

        public MainPage() {
            InitializeComponent();
        }

        #region #AnimationConfig
        protected override void OnLoad(EventArgs e) {
            base.OnLoad(e);

            view.Animation = new FanFunnelAnimation {
                BeginTime = new TimeSpan(0, 0, 0),
                Duration = new TimeSpan(0, 0, 0, 1, 500),
                PointDelay = new TimeSpan(0, 0, 0, 0, 500),
                PointOrder = PointAnimationOrder.Straight,
                EasingFunction = new LinearEasingFunction {
                    EasingMode = EasingMode.Out
                }
            };

            chartControl.AnimationStartMode = ChartAnimationMode.OnLoad;
        }

        private void OnAnimateClick(object sender, EventArgs e) {
            chartControl.Animate();
        }
        #endregion #AnimationConfig
    }
}

File: VB/AnimationExample/MainPage.vb
Imports DevExpress.XtraCharts
Imports System

Namespace AnimationExample

    Public Partial Class MainPage
        Inherits DevExpress.XtraEditors.XtraForm

        Private ReadOnly Property series As Series
            Get
                Return chartControl.Series(0)
            End Get
        End Property

        Private ReadOnly Property view As FunnelSeriesView
            Get
                Return TryCast(series.View, FunnelSeriesView)
            End Get
        End Property

        Public Sub New()
            InitializeComponent()
        End Sub

#Region "#AnimationConfig"
        Protected Overrides Sub OnLoad(ByVal e As EventArgs)
            MyBase.OnLoad(e)
            view.Animation = New FanFunnelAnimation With {.BeginTime = New TimeSpan(0, 0, 0), .Duration = New TimeSpan(0, 0, 0, 1, 500), .PointDelay = New TimeSpan(0, 0, 0, 0, 500), .PointOrder = PointAnimationOrder.Straight, .EasingFunction = New LinearEasingFunction With {.EasingMode = EasingMode.Out}}
            chartControl.AnimationStartMode = ChartAnimationMode.OnLoad
        End Sub

        Private Sub OnAnimateClick(ByVal sender As Object, ByVal e As EventArgs)
            chartControl.Animate()
        End Sub
#End Region  ' #AnimationConfig
    End Class
End Namespace
